lib/ex338_web.ex

```diff
@@ -74,7 +75,7 @@ defmodule Ex338.Web do
       alias Ex338.Repo
       import Ecto
       import Ecto.Query
-      import Ex338.Gettext
+      import Ex338Web.Gettext
```

Comment:
Most of the time you are using the multi-alias/require/import/use syntax, but here you are using multiple single directives

Reply:
ok